src/Common/src/Interop/Unix/libcurl/Interop.libcurl_types.cs

```diff
@@ -12,8 +14,14 @@ internal static partial class CURLoption
             private const int CurlOptionLongBase = 0;
             private const int CurlOptionObjectPointBase = 10000;
 
-            internal const int CURLOPT_FOLLOWLOCATION = CurlOptionLongBase + 52;
             internal const int CURLOPT_URL = CurlOptionObjectPointBase + 2;
```

Comment:
The options are in increasing order of value. 
 The above line was from the old version :-)  The change introduced  is the below snippet..